babel.config.json

```diff
@@ -5,7 +5,7 @@
 			{
 				"targets": "defaults, node >= 12",
 				"useBuiltIns": "usage",
-				"corejs": "3.25"
+				"corejs": "3.30"
```

Comment:
Updated to match the current installed version, as [recommended by Babel](https://babeljs.io/docs/babel-preset-env#corejs).